setup.py

```diff
@@ -79,6 +74,7 @@
         'jsonschema>=3.2.0',
         'jupyter_core>=4.0,<5.0',
         'jupyter_client>=6.1.7',
+        'jupyter-packaging>=0.10,<2',
```

Comment:
Is this a runtime dependency now?

Can we please remove the cap (per the previous comment)?